Keas.Mvc/ClientApp/components/Equipment/EquipmentBigFixSearchId.tsx

```diff
@@ -0,0 +1,280 @@
+                    <FormFeedback>Computer name is required</FormFeedback>
+                </FormGroup>
+
+                {this._renderNameTable()}
+            </Form>
+        );
+    };
+
+    private _renderInputSearch = () => {
+        if (this.state.selectedFeild === "Name") {
+            return (
+                <>
+                    <label>Name</label>
+                    <Input
+                        type="text"
+                        name="name"
+                        id="computer name"
+                        placeholder="Enter Computer Name"
+                        invalid={this.state.valueToBeSearched.length < 1}
+                        onChange={e => {
+                            this.setState({
+                                valueToBeSearched: e.target.value
+                            });
+                        }}
+                    />
+                </>
+            );
+        } else if (this.state.selectedFeild === "Id") {
+            return <Input type="text" name="Id" id="computer Id" placeholder="Enter Computer Id" />;
+        } else if (this.state.selectedFeild === "Company") {
+            return (
+                <Input
+                    type="text"
+                    name="Company"
+                    id="computer Company"
+                    placeholder="Enter Computer Company"
+                />
+            );
+        }
+    };
+
+    private _renderNameTable = () => {
+        if (!this.state.isFetched) {
+            return null;
+        }
+
+        if (!this.state.isValidSearch) {
+            return (
+                <p className="text-center text-danger">
+                    No computer found by this name, please try again.
+                </p>
+            );
+        }
+        return (
+            <Table>
+                <tbody>
+                    {this.state.listOfComputers.map(computer => {
+                        return (
+                            <tr
+                                className="bigfix-info"
+                                onClick={() =>
+                                    this.props.addBigFixId("systemManagementId", computer.id)
+                                }
+                                key={computer.id}
+                            >
+                                <td>{computer.name}</td>
+                            </tr>
+                        );
+                    })}
+                </tbody>
+            </Table>
+        );
+    };
+
+    private _renderSearchButton = () => {
+        if (this.state.isSearching) {
+            return (
+                <Button color="primary" disabled={false}>
+                    {" "}
+                    <i className="fas fa-lg fa-spinner fa-pulse" />
+                </Button>
+            );
+        } else {
+            return (
+                <Button
+                    color="primary"
+                    disabled={false}
+                    onClick={() => {
```

Comment:
My preference is, if the onClick is more than one line/statement, move it into a new method, like `onSearch` or w/e